informatiemodel/rdf/ontologie.ttl

```diff
@@ -801,13 +864,21 @@ zvg:Legitimatie.identificatie a sh:PropertyShape;
 .
 zvg:LijstVanGoederen a <http://www.w3.org/2002/07/owl#Class>, sh:NodeShape;
     sh:name "LijstVanGoederen";
+    sh:property zvg:LijstVanGoederen.identificatie;
     sh:property zvg:LijstVanGoederen.isOnderdeelVan;
     sh:property zvg:LijstVanGoederen.lijstReferentie;
     sh:property zvg:LijstVanGoederen.waarde;
     sh:targetClass zvg:LijstVanGoederen;
     dct:subject <http://mak.zorgeloosvastgoed.nl/id/concept/LijstVanGoederen>;
     rdfs:label "LijstVanGoederen"
 .
+zvg:LijstVanGoederen.identificatie a sh:PropertyShape;
+    sh:datatype xsd:string;
+    sh:maxCount "1";
+    sh:minCount "1";
+    sh:name "identificatie";
```

Comment:
voorstel: in het informatiemodel aangevel dat modelelementen in de nederlandse taal zijn weergegeven: https://docs.geostandaarden.nl/mim/mim/#metagegeven-mim-taal

Wordt dit dan ook gebruikt tijdens de transformatie?